.gitignore

```diff
@@ -120,6 +120,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
+.envrc
```

Comment:
Do we need this? it's already listed in line 125

Reply:
Good catch